skills_ml/algorithms/occupation_classifiers/classifiers.py

```diff
@@ -1,5 +1,11 @@
 from skills_ml.algorithms.embedding.base import ModelStorage
-from skills_ml.algorithms.embedding.models import Doc2VecModel
+from skills_ml.algorithms.embedding.models import Doc2VecModel, EmbeddingTransformer
+from skills_ml.algorithms.occupation_classifiers.train import SocEncoder
+from skills_ml.ontologies.onet import majorgroupname
+
+from sklearn.base import BaseEstimator, ClassifierMixin
+from sklearn.utils.validation import check_X_y, check_array, check_is_fitted
+from sklearn.pipeline import make_pipeline, Pipeline
```

Comment:
Could clean up the imports. I was curious about some of these imports and Ctrl+F'd them to no avail.